src/types/Namespace.ts

```diff
@@ -3,3 +3,11 @@ interface Namespace {
 }
 
 export default Namespace;
+
+export const namespaceFromString = (namespace: string) => ({ name });
```

Comment:
Just out of curiosity, why do we even have Namespace type as an object with only a name field?  It's sort of a pain, I don't see the value of it over just being a string.

Reply:
Me neither, I think it was just in case we needed to add more data to it?
 @mtho11 do you know?

Reply:
If no one knows then I'm not against changing it to a string and a string[] for namespaces.